src/ITILFollowup.php

```diff
@@ -890,6 +890,13 @@ public static function showMassiveActionsSubForm(MassiveAction $ma)
         return parent::showMassiveActionsSubForm($ma);
     }
 
+    /**
+     * @see CommonDBTM::processMassiveActionsForOneItemtype()
```

Comment:
Check item type inside method and add KO status for all IDs if it isn't CommonITILObject.